src/status_im/ui/screens/events.cljs

```diff
@@ -294,7 +294,9 @@
               status-module-initialized? status-node-started? device-UUID]
        :or   [network (get app-db :network)]} [_ address]]
    (let [console-contact (get contacts constants/console-chat-id)
-         current-account (accounts address)]
+         current-account (accounts address)
```

Comment:
better to use `get` , `(get accounts address)`

Reply:
@flexsurfer yes, but decided not to change it as this code is not changed in current PR.

Reply:
oh sorry ,thought you added this line